src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -1109,6 +1126,58 @@ private void AddPatchNode(Patch patch, Vector2 position)
 
         patch.ApplyPatchEventVisuals(node);
 
+        var playerSpecies = patch.FindSpeciesByID(PlayerSpeciesID);
+        if (playerSpecies != null)
+        {
+            var playerPopulationIndicatorAmount = (int)Math.Ceiling(
+                patch.GetSpeciesSimulationPopulation(playerSpecies) * 0.004);
+
+            var indicatorExcess = Math.Clamp(playerSpeciesPopulationIndicators.Count - playerPopulationIndicatorAmount,
+                0,
+                playerSpeciesPopulationIndicators.Count);
+
+            for (int i = 0; i < indicatorExcess; ++i)
+            {
+                playerSpeciesPopulationIndicators.Last().QueueFree();
```

Comment:
I think it would be better to just hide the excess indicators, I don't expect us to have so many excess ones that they'd cause an issue during the lifetime of the map drawer node.